src/node/install.sh

```diff
@@ -16,6 +16,9 @@ INSTALL_TOOLS_FOR_NODE_GYP="${NODEGYPDEPENDENCIES:-true}"
 # alongside NODE_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
+#pnpm Version
+PNPM_VERSION="${PNPMVERSION:-"7"}"
```

Comment:
```suggestion
PNPM_VERSION="${PNPMVERSION:-"latest"}"
```

Let's install latest version when use does not pass a version

Reply:
done